napari/_qt/layer_controls/qt_points_controls.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, layer):
         )
         sld.setFocusPolicy(Qt.FocusPolicy.NoFocus)
         sld.setMinimum(1)
-        sld.setMaximum(100)
+        sld.setMaximum(max(100, int(np.max(self.layer.size)) + 1))
```

Comment:
This will not work for layers with zero points. 

```
ValueError: zero-size array to reduction operation maximum which has no identity
```

Reply:
I also just realised and I provided a fix for that